managed/ui/src/components/config/Storage/BackupList.js

```diff
@@ -0,0 +1,196 @@
+        </Button>
+      </FlexShrink>
+    </FlexContainer>
+  </>
+);
+
+export const BackupList = (props) => {
+  const [configData, setConfigData] = useState({});
+  const [associatedUniverses, setUniverseDetails] = useState([]);
+  const [isUniverseVisible, setIsUniverseVisible] = useState(false);
+  const {
+    activeTab,
+    data,
+    deleteStorageConfig,
+    hideDeleteStorageConfig,
+    onCreateBackup,
+    onEditConfig,
+    showDeleteStorageConfig,
+    visibleModal
+  } = props;
+  const currTab = activeTab.toUpperCase();
+
+  // This method will handle all the required actions for
+  // the particular row.
+  const formatConfigActions = (cell, row) => {
+    const {
+      configName,
+      configUUID,
+      inUse,
+      name,
+      universeDetails
+    } = row;
+
+    return (
+      <>
+        <DropdownButton
+          className="backup-config-actions btn btn-default"
+          title="Actions"
+          id="bg-nested-dropdown"
+          pullRight
+        >
+          <MenuItem onClick={() => onEditConfig(row)}>
+            <i className="fa fa-pencil"></i> Edit Configuration
+          </MenuItem>
+          <MenuItem
+            disabled={inUse}
+            onClick={() => {
+              setConfigData(configUUID);
+              showDeleteStorageConfig(name);
+            }}
+          >
+            {!inUse && (
+              <>
+                <i className="fa fa-trash"></i> Delete Configuration
+              </>
+            )}
+
+            {inUse && (
+              <YBInfoTip
+                content="Storage configuration is in use and cannot be deleted until associated resources are removed."
+                placement="top"
+              >
+                <span className="disable-delete">
+                  <i className="fa fa-ban"></i> Delete Configuration
+                </span>
+              </YBInfoTip>
+            )}
+          </MenuItem>
+
+          {
+            <YBConfirmModal
+              name="delete-storage-config"
+              title="Confirm Delete"
+              onConfirm={() => deleteStorageConfig(configData)}
+              currentModal={'delete' + name + 'StorageConfig'}
+              visibleModal={visibleModal}
+              hideConfirmModal={hideDeleteStorageConfig}
+            >
+              Are you sure you want to delete {name} Storage Configuration?
+            </YBConfirmModal>
+          }
+
+          <MenuItem
+            onClick={() => {
+              setConfigData(configName);
+              setUniverseDetails([...universeDetails]);
+              setIsUniverseVisible(true);
+            }}
+          >
+            <i className="fa fa-eye"></i> Show Universes
+          </MenuItem>
+        </DropdownButton>
+        <AssociatedUniverse
+          visible={isUniverseVisible}
+          onHide={() => setIsUniverseVisible(false)}
+          associatedUniverses={associatedUniverses}
+          title={`Backup Configuration ${configData}`}
+        />
```

Comment:
Please move the `AssociatedUniverse` out of the `formatConfigActions()` function. This modal can be included after the `BootstrapTable`.

Reply:
Done.